ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/139__trusted_ca_bundles.robot

```diff
@@ -0,0 +1,155 @@
+*** Settings ***
+Documentation    Test Cases to verify Serverless installation
+Library    Collections
+Resource    ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Suite Setup    Suite Setup
+Suite Teardown    Suite Teardown
+
+
+*** Variables ***
+${RHOAI_OPERATOR_NS}    redhat-ods-operator
+${RHOAI_OPERATOR_DEPLOYMENT_NAME}    rhods-operator
+${TEST_NS}    test-trustedcabundle
+${DSCI_NAME}    default-dsci
+${TRUSTED_CA_BUNDLE_CONFIGMAP}    odh-trusted-ca-bundle
+${CUSTOM_CA_BUNDLE}    test-example-custom-ca-bundle
```

Comment:
could it be possible to use a real case value? you could generate a certificate? we have a keyword which does it, could you check if it fits this case? 
`Generate Client TLS Certificates` from `ods_ci/tests/Resources/CLI/ModelServing/llm.resource`

Reply:
Will investigate how to do as suggested, which would be a nice improvement.